docs/sources/flow/reference/components/loki.source.kubernetes_events.md

```diff
@@ -152,6 +152,21 @@ events in each watched namespace.
 
 `loki.source.kubernetes_events` does not expose any component-specific debug metrics.
 
+## Component behavior
+
+The component uses its data path (a directory named after the domain's
+fully qualified name) to store its _positions file_. The positions file is used
+to store read offsets, so that in case of a component or Agent restart,
+`loki.source.kubernetes_events` can pick up tailing from the same spot.
```

Comment:
```suggestion
The component uses its data path,  a directory named after the domain's
fully qualified name, to store its _positions file_. The positions file is used
to store read offsets, so that if a component or {{< param "PRODUCT_ROOT_NAME" >}} restarts,
`loki.source.kubernetes_events` can pick up tailing from the same spot.
```